### PR TITLE
Filter cp outputs

### PR DESCRIPTION
**Pull request by ntrncic**
connects unicef/etools-issues#26
connects unicef/etools-issues#8

### CODE REVIEW

**Comment by robertavram** [comment]
@simon-the-sloth is nico using this endpoint to get CP Outputs... if so.. will this filter change have an impact?

File: EquiTrack/reports/views/v2.py [hunk 1]
```diff
@@ -25,20 +25,44 @@ def get_queryset(self):
...
-                cp_year = query_params.get("country_programme", None)
+            if "year" in query_params.keys():
+                cp_year = query_params.get("year", None)
                 queries.append(Q(country_programme__wbs__contains='/A0/'))
                 queries.append(Q(country_programme__from_date__year__lte=cp_year))
                 queries.append(Q(country_programme__to_date__year__gte=cp_year))
             if "result_type" in query_params.keys():
                 queries.append(Q(result_type__name=query_params.get("result_type")))
-
+            if "country_programme" in query_params.keys():
```
Comment (robertavram) [inline EquiTrack/reports/views/v2.py]: @ntrncic letting you figure this out

**Comment by adi130987** [comment]
@ntrncic `GET /api/v2/reports/results?values=2` is returning all cp output object properties:

```javascript
[
  {
    "id": 2,
    "result_type": "Output",
    "name": "CP & GBV VIOLATIONS ARE MITIGATED",
    "code": null,
    "from_date": "2013-01-02",
    "to_date": "2016-12-31",
    "humanitarian_tag": false,
    "wbs": "2490/A0/06/102/010",
    "vision_id": null,
    "gic_code": null,
    "gic_name": null,
    "sic_code": null,
    "sic_name": null,
    "activity_focus_code": null,
    "activity_focus_name": null,
    "hidden": false,
    "ram": true,
    "lft": 2,
    "rght": 11,
    "tree_id": 1,
    "level": 1,
    "result_structure": null,
    "country_programme": 1,
    "sector": null,
    "parent": 1
  }
]
```

I only need `id`, `name` and `wbs`. can this be done? or it will have impact on other functionalities as well.

**Comment by ntrncic** [comment]
@adi130987 use parameter dropdown=true that will return only id, name, wbs

**Comment by adi130987** [comment]
`GET /users/api?values=missingUserIds` works fine, but I have an issue because of the data returned by `api/v2/dropdowns/pmp`, each `signed_by_unicef_users` looks like this:

```javascript
{
      "first_name": "Adrian",
      "last_name": "Hangan",
      "id": 18314,
      "email": "adrian.hangan@nordlogic.com"
}
```
I need that `signed_by_unicef_users` from `dropdowns/pmp` endpoint to have same properties as `/users/api` returns:

```javascript
{
    "user_id": "18314",
    "full_name": "Adrian Hangan",
    "username": "adi",
    "email": "adrian.hangan@nordlogic.com"
}
```

**Comment by ntrncic** [comment]
@adi130987 Can't you split the name?

**Comment by ntrncic** [comment]
@adi130987 changed the pmp dropdown api to return the same as users/api/

**Comment by adi130987** [comment]
Thanks @ntrncic. Frontend work completed https://github.com/unicef/etools-partnership-management/pull/222

**Pull request merged**